client/common/directives/ModelField/ModelField.js

```diff
@@ -148,7 +148,7 @@ angular.module('dashboard.directives.ModelField', [
         template = '<label class="col-sm-2 control-label">{{ display.label || key }}:</label>\
           <div class="col-sm-10 multi-select">\
             <div class="select-item checkbox-container" ng-repeat="(itemKey, itemValue) in display.options">\
-              <input type="checkbox" class="field" my-space="clickMultiSelectCheckbox(key, itemKey, itemValue, multiSelectOptions)" ng-model="multiSelectOptions[itemKey]" ng-checked="multiSelectOptions[itemKey]" ng-disabled="display.readonly">\
+              <input type="checkbox" class="field" ng-click="clickMultiSelectCheckbox(key, itemKey, itemValue, multiSelectOptions)" ng-model="multiSelectOptions[itemKey]" ng-checked="multiSelectOptions[itemKey]" ng-disabled="display.readonly">\
```

Comment:
Does this still allow the TAB_SPACE behavior to work?

If we don't need the my-space directive anymore, we should remove it.